gemrb/GUIScripts/pst/NewLife.py

```diff
@@ -250,17 +250,13 @@ def AcceptPress():
 
 	#don't add con bonus, it will be calculated by the game
 	#interestingly enough, the game adds only one level's con bonus
-	Con = Stats[4]
-	if Con > 14:
-		x = 30
-	else:
-		x = 20 + (Con-9)*2
+	Table = GemRB.LoadTable ("hpinit")
+	x = Table.GetValue (str(Stats[4]), "HP")
 
 	print("Setting max hp to: ", x)
 	GemRB.SetPlayerStat(1, IE_MAXHITPOINTS, x)
-	#adding the remaining constitution bonus to the current hp
-	#if Con>14:
-	#	x = x+(Con-14)*3
+	
+	x = x + 100 #ensures the player starts at max hp when con > 14
```

Comment:
Do you want me to remove this line completely as well 

or just make it
`GemRB.SetPlayerStat(1, IE_HITPOINTS, x + 100)`

Reply:
If you do that, also remove the print or they won't match. Don't have a preference either way.

Reply:
Ok, the print for setting max hp is also wrong if con > 14. Should I remove that as well?

Reply:
Ah, true — best to remove both then, yeah.

Reply:
Done!